app/components/documents/document-card.hbs

```diff
@@ -41,7 +41,7 @@
               {{! no piece.created in legacy data }}
               {{#if this.piece.created}}
                 <p data-test-document-card-primary-source-created={{if this.piece.file.derived ""}}>
-                  {{t "uploaded-at"}}
+                  {{t this.label}}
```

Comment:
Works fine like this, however I would move the translation into the `label` getter, this way the name `label` better reflects its contents.